Modules/mathmodule.c

```diff
@@ -759,13 +759,20 @@ m_log10(double x)
 static PyObject *
 math_gcd(PyObject *module, PyObject * const *args, Py_ssize_t nargs)
 {
-    PyObject *res, *x;
-    Py_ssize_t i;
+    // Fast-path for the common case: avoid calling _PyNumber_Index()
+    // and the loop.
+    if (nargs == 2
+        && PyLong_CheckExact(args[0])
+        && PyLong_CheckExact(args[1]))
+    {
```

Comment:
It could fit in one line, no?

Reply:
I changed the code to fit into a single line.